docs/teamresources/guidelines.md

```diff
@@ -46,7 +45,7 @@ var ErrSentinel = errors.New("identifiable error")
 // turn anonymous 3rd party errors into identifiable ones
 func SentinelErrFrom3rdParty() (result interface{}, err error)
     if result, err = 3rdPartyLibrary.DoSomething(); err != nil {
-        err = xerrors.Errorf("failed to do something (%v): %w", err, ErrSentinel)
+        err = errors.Errorf("failed to do something (%v): %w", err, ErrSentinel)
```

Comment:
What's that red `%`?

Reply:
I guess it's because `guidelines.md` isn't a `*.go` file and this how GitHub displays it. In all other places in `*.go` files, there is no red thing.